typings/index.d.ts

```diff
@@ -3114,6 +3115,29 @@ declare module 'discord.js' {
 
   type Status = number;
 
+  export class Sticker extends Base {
+    constructor(client: Client, data: object);
+    public asset: string;
+    public readonly createdTimestamp: number;
+    public readonly createdAt: Date;
+    public description: string;
+    public format: StickerFormatTypes;
+    public id: Snowflake;
+    public name: string;
+    public packID: Snowflake;
+    public previewAsset: string | null;
+    public tags: string[] | null;
+    public stickerURL(options?: StickerURLOptions): string | null;
```

Comment:
```suggestion
    public stickerURL(options?: StickerURLOptions): string;
```
this always returns a string